lib/register/transform.js

```diff
@@ -23,9 +23,9 @@ var transformTypes = ['name', 'value', 'attribute'];
  * @function
  * @param {Object} transform - Transform object
  * @param {String} transform.type - Type of transform, can be: name, attribute, or value
- * @param {String} transform.name - Name of the transformer so a transformGroup can call a list of transforms.
+ * @param {String} transform.name - Name of the transformer by which a transformGroup can call a list of transforms.
```

Comment:
Name of the transformer. (A transformGroup uses these names to create a list of transforms)